### DISCUSSION TITLE
Pong Tutorial 03 - Scripting the Game Start Menu

### DISCUSSION THREAD

**Discussion by laserbeak** [o3de/o3de#9354] [Help Wanted]
Hi,  when I script the pong tutorial,  I don't get any output in the editor.
[Youtube Link](https://youtu.be/6-353OtcTkM?t=356)
I've tried adding the onGraphStart node as suggested in the youtube comments, but that didn't work either.

![image](https://user-images.githubusercontent.com/3384800/166401975-cec14dc2-a0ce-4860-9cb5-520dcc3747f4.png)
![image](https://user-images.githubusercontent.com/3384800/166402030-54015533-85ae-4d82-a90f-3e0cb8e30a18.png)

**Comment by tarboss** [comment]
- start in Script Canvas with Game Enitity -> InputHandler -> Print for output
- look at my screen shot for details - worked with Developer Preview version (source compiled by me)
![Pong3d_startmenu](https://user-images.githubusercontent.com/30550163/167673462-c3917005-9292-4d89-b717-c9883203a0bb.jpg)

**Reply by Saintyboy** [reply]
I am stumped with this same issue - nothing prints to the console - I am going to continue and hope that the proper functionality works - if it does then it would seem like there is some issue with the print to console node or maybe something is missing. @tarboss  your screen shot is with an older version - our newer versions have extra fields in the Game entity node and i reckon its something to do with that.

**Reply by tarboss** [reply]
try with 2 nodes: GameEnity and Print Debug. U can see my console output if u press play. If thats not happen, something not working with ur version or some console settings.
![o3de_console](https://user-images.githubusercontent.com/30550163/171115727-aed01218-ae57-441c-95cb-7d37eb2b9158.jpg)
